pipeline/alignments/generate-alignment-merged.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
I know you had a justification for not switching this to python, but it really does feel consequential enough changes to benefit from a switch. This essentially adding to technical debt at this point.